contracts/translation/translation.go

```diff
@@ -0,0 +1,24 @@
+package translation
+
+//go:generate mockery --name=Translation
+type Translation interface {
+	// Load translations from path.
+	Load(path string, locale ...string) error
+	GetDefaultLocale() string
+	//
+	Language(locale string) Language
+	// Add translations to application.
+	Add(name, locale string, val any)
+	// Get translation from application.
+	Get(path string, locale string, options ...any) string
```

Comment:
S: We can add a `Option` struct, just like: [validation/options.go](https://github.com/goravel/framework/blob/master/validation/options.go)
```suggestion
	Get(path string, options ...Option) string
```